examples/move/first_package/sources/example.move

```diff
@@ -112,7 +112,7 @@ module first_package::example {
             ts::return_to_sender(&ts, forge);
         };
 
-        // third transaction executed by the initial sword owner
+        // third transaction executed by the ALICE
```

Comment:
@qianyiwangmeng, can you explain the motivation for this change?  The comment as previously written was correct ("ALICE" is the name of the initial sword owner) so I want to understand if the wording was in some way confusing.

The current change makes the comment redundant because you can tell from the first line of each block who is executing the transaction (there is also a small grammatical typo:)

```suggestion
        // third transaction executed by ALICE
```

Also note that if we want to make this change, we also need to:

- Make a similar change to the comment for the last transaction.
- Update the docs at `docs/content/guides/developer/first-app/build-test.mdx` which references this code.

Reply:
Thanks. I previously think "ADMIN" is the name of the initial sword owner. It makes me confused.

Reply:
In that case, let's make the comment "third transaction executed by ALICE, the initial sword owner"!